### PR TITLE
Altering the repo name to be a link that takes you to the Docker Hub page

### PR DESCRIPTION
**Pull request by allen-servedio**
Altered the repo name on the ImageCard to open the Docker Hub page for the repo when it is clicked on. I tried to follow the style that you all use for the tag link - though I am not sure that is how you want it to be styled (feel free to change to conform to the standards you are trying to set).

A more advanced version of this might be to open that page within another window.

### CODE REVIEW

**Comment by DieterReuter** [comment]
Just tested it, works great. :+1:
But the graphical response distracts me a little, it overlaps the text way too much. Maybe we have an UX designer here, who could help to get the best visual effect.

@jeffdm: Is it possible to open such an external web link in-place in the apps view? Sometimes it's really annoying, when you'll get a new browser windows for each and every click. I just would like to see both ways side-by-side, to come to a better decision what's more convenient for the users.

**Comment by allen-servedio** [comment]
Yep, I agree with you. It might be a good idea to form up a basic style guide or something so that contributions remain consistent with the over all UX experience.  I think having the page appear as a window in the app makes sense (the overviews are just not enough info for people to pick a container). I am also thinking of adding this to running containers for folks to be able to see what the container is based on.

**Comment by Elesant** [comment]
Thank you so much for this! Definitely a great feature. Will figure out a design and style guide that makes sense. :+1:

**Comment by allen-servedio** [comment]
Great and thanks. If you all are open to it, I will try to add things as I hit them. For example,  I may take a swag at pulling the dockerfile and let them view it right in the app. Developers like details, and this seems like a logical thing to display.

**Comment by Elesant** [comment]
@allen-servedio I think that's a pretty cool idea. It will definitely improve the "content" side of the design for developers. Imagine you can view a well formatted and syntax highlighted Dockerfile right from the app. It would be nice if we can figure out a way together to keep things super simple while are introducing new contents as well :)

**Comment by jeffdm** [comment]
Awesome. Taking a look!

File: styles/new-container.less [hunk 1]
```diff
@@ -186,6 +186,12 @@
           overflow: hidden;
           text-overflow: ellipsis;
           white-space: nowrap;
+          text-decoration: underline;
+          &:hover {
```
Comment (jeffdm) [inline styles/new-container.less]: Can we remove the hover-over styling for now?

File: src/ImageCard.react.js [hunk 2]
```diff
@@ -67,7 +77,7 @@ var ImageCard = React.createClass({
         <div>
           <div className="namespace">{namespace}</div>
           <OverlayTrigger placement="bottom" overlay={<Tooltip>{this.props.image.name}</Tooltip>}>
```
Comment (jeffdm) [inline src/ImageCard.react.js]: Since it's redundant, should we change the Tooltip text to `View on DockerHub`?

File: src/ImageCard.react.js [hunk 3]
```diff
@@ -40,6 +41,15 @@ var ImageCard = React.createClass({
     var $tagOverlay = $(this.getDOMNode()).find('.tag-overlay');
     $tagOverlay.fadeOut(300);
   },
+  handleRepoClick: function () {
+    var $repoUri = 'https://registry.hub.docker.com/'
```
Comment (jeffdm) [inline src/ImageCard.react.js]: + add semicolons

**Comment by jeffdm** [comment]
Awesome! This is really cool (and asked for a lot). I added some comments

**Comment by allen-servedio** [comment]
Sorry for the delay - I made the changes you suggested. Let me know if you want anything else done to it. Thanks for considering the change!

**Comment by jeffdm** [comment]
LGTM, merging!

**Comment by jeffdm** [comment]
Thanks for this! Awesome!

**Comment by allen-servedio** [comment]
Hey Jeff - I goofed. I missed one of the tool tips (sorry about that). I pushed another commit to fix it (it is just below the one I fixed).

**Comment by jeffdm** [comment]
No worries! I'll pull it in.

Thanks again @allen-servedio !

**Comment by jeffdm** [comment]
Ok. Merged! Great! Thanks @allen-servedio !

@DieterReuter this is a neat idea, showing web pages in-app vs opening a separate browser window. We'd need some sort of navigation tree built-in, similar to iOS & Android apps. cc @Elesant

**Pull request merged**